CONTRIBUTING.md

```diff
@@ -441,10 +441,27 @@ The API reference is manually assembled in `doc/api/index.rst`.
 The *autodoc* sphinx extension will automatically create pages for each
 function/class/module listed there.
 
-You can reference classes, functions, and modules from anywhere (including docstrings)
-using <code>:func:\`package.module.function\`</code>,
-<code>:class:\`package.module.class\`</code>, or
-<code>:mod:\`package.module\`</code>.
+You can reference classes, functions, methods, and modules from anywhere 
+(including docstrings) using <code>:func:\`package.module.function\`</code>,
+<code>:class:\`package.module.class\`</code>, 
+<code>:meth:\`package.module.method\`</code>, or
+<code>:mod:\`package.module\`</code>. An example would be to use 
```

Comment:
In addition to the comment at https://github.com/GenericMappingTools/pygmt/pull/802/files#r580581522, I think it would look nicer to break this long paragraph into a bulleted list.

```suggestion
(including docstrings) using:

- <code>:func:\`package.module.function\`</code>
- <code>:class:\`package.module.class\`</code>
- <code>:meth:\`package.module.method\`</code>
- <code>:mod:\`package.module\`</code>

An example would be to use 
```

Maybe also use a bulleted list for `:gmt-docs:` and `:gmt-term:` below.